oauth_dropins/meetup.py

```diff
@@ -0,0 +1,162 @@
+"""Meetup.com drop-in.
+
+API docs:
+https://www.meetup.com/meetup_api/
+"""
+import logging
+import urllib.parse
+
+from google.cloud import ndb
+from webob import exc
+
+from . import handlers
+from .models import BaseAuth
+from .webutil import appengine_config, appengine_info, util
+from .webutil.util import json_dumps, json_loads
+
+if appengine_info.DEBUG:
+    MEETUP_CLIENT_ID = util.read('meetup_client_id_local')
```

Comment:
do meetup's app settings let you specify more than one redirect URL per app? if so, feel free to drop the local client id/secret files, we won't need them.

Reply:
Unfortunately not, it appears to only want one `redirect_uri` - so I'll leave this in here